packages/office-ui-fabric-react/src/components/TextField/TextField.scss

```diff
@@ -187,19 +187,13 @@ $field-error-color: $errorTextColor;
 
 //== Modifier: Single line (default), underlined
 .root.rootIsUnderlined {
-  border-width: 0px;
-
-  &:hover:not(.rootIsDisabled) {
-    border-color: $field-border-hover-color;
-
-    @include high-contrast {
-      border-color: Highlight;
-    }
-  }
+  border: 0px solid $field-border-color;
 
   .wrapper {
     display: flex;
-    border-bottom: 1px solid $field-border-color;
+    border-bottom-width: 1px;
+    border-bottom-style: solid;
+    border-bottom-color: inherit;
```

Comment:
make sure the "inherit" works in ie11

Reply:
@phkuo it does work in IE11, it just needs to be split into width/style/color instead of using the border shorthand property.